index.js

```diff
@@ -47,6 +47,12 @@ async function run() {
 
         const {stdout} = await subprocess(["crystal", "--version"]);
         Core.info(stdout);
+
+        if (Core.getBooleanInput("annotate")) {
+            const matchersPath = Path.join(__dirname, ".github");
+            Core.info(`##[add-matcher]${Path.join(matchersPath, "crystal.json")}`);
+            Core.info(`##[add-matcher]${Path.join(matchersPath, "crystal-spec.json")}`);
```

Comment:
I'm surprised that this old-style GitHub syntax actually works :o

But I think this would work better and also emit the new-style syntax `::add-matcher`

```suggestion
            Core.issueCommand("add-matcher", {}, Path.join(matchersPath, "crystal.json"));
            Core.issueCommand("add-matcher", {}, Path.join(matchersPath, "crystal-spec.json"));
```

https://github.com/actions/toolkit/blob/master/docs/commands.md#problem-matchers

Reply:
It doesn't look like that function is currently exported and available for use :(

I've just popped open [an issue](https://github.com/actions/toolkit/issues/853) for that and switched across to the `::` syntax in 8340c7c.